i18n/nl/docusaurus-plugin-content-docs/current/staff/coaches/attendance.md

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 3
+---
+
+# Aanwezigheid
+
+Aanwezigheidsregistratie in Jarvis is een optionele functie die in 2 verschillende modi kan functioneren.
+
+Beide modi tonen de student een grafiek op hun dashboard van de geregistreerde aanwezigheid.
+
+![dashboard widget voor studentenaanwezigheid](/img/coaches/attendance/student-dashboard.png)
+
+Daarnaast, als de chatfunctie is ingeschakeld, ontvangt de student geautomatiseerd chatbericht als
+ze worden ziek, te laat of ongeoorloofd afwezig geregistreerd gemeld worden.
+
```

Comment:
Ik mis nog iets over:
- hoelang is aanwezigheid te wijzigen (altijd) 
- dat het wordt meegenomen in de locatie gezondheid
- dat het meteen wordt opgeslagen (en dus zichtbaar is voor de student) als je het aanklikt in jarvis. Dus niet apparte opslaan knop oid)
- Dat aanwezigheid per dag werkt, niet per uur. 
- Hoe ze studenten uit het aanwezigheidsoverzicht krijgen (geblokkeerd, of status op stage / afgestudeerd)